pyaedt/edb_core/padstack.py

```diff
@@ -496,8 +500,12 @@ def create_coax_port(self, padstackinstance, use_dot_separator=True):
             port_name = "{0}_{1}_{2}".format(cmp_name, pin_name, net_name)
         if not padstackinstance.IsLayoutPin():
             padstackinstance.SetIsLayoutPin(True)
-
         res = padstackinstance.GetLayerRange()
+        if name:
+            if not self._port_exist(name):
+                port_name = name
+        else:
+            self._logger.info("Port already defined with this name, using default naming convention")
```

Comment:
Recommend changing the error message to be more consistent with the earlier one: An existing port already has this same name. Using the default naming convention.